src/main/java/response/HttpResponseWriter.java

```diff
@@ -0,0 +1,20 @@
+package com.td.HttpServer;
+import java.io.IOException;
+
+public class HttpResponseWriter {
```

Comment:
Yeah, if it's simple delegation it's hard to justify _unless_ you're delegating to things you don't own (i.e. file system calls, external service calls, other-library calls).